react-bootstrap-table/react-bootstrap-table.d.ts

```diff
@@ -520,11 +558,13 @@ declare module "react-bootstrap-table" {
 		options?: any;
 	}
 	
+	export type FilterType = 'TextFilter' | 'RegexFilter' | 'SelectFilter' | 'NumberFilter' | 'DateFilter' | 'CustomFilter';
```

Comment:
How can users add their custom filters? Is there a way to make this type extensible?